src/stores/Entities/hooks.ts

```diff
@@ -12,24 +12,33 @@ import { useEntitiesStore } from './Store';
 //  I think there is a chance we could use some kind of permissions library to help with this.
 
 export const useEntitiesStore_capabilities_readable = () => {
-    return useEntitiesStore((state) => ({
-        ...state.capabilities.admin,
-        ...state.capabilities.write,
-        ...state.capabilities.read,
-    }));
+    return useEntitiesStore(
+        useShallow((state) =>
+            Object.keys({
+                ...state.capabilities.admin,
+                ...state.capabilities.write,
+                ...state.capabilities.read,
+            })
+        )
+    );
```

Comment:
We were always using `Object.keys` so I think we can pull this into the store and stop repeating that code so much.

Also adding shallow.